### PR TITLE
added common interface to cornflow

### PR DESCRIPTION
**Pull request by pchtsp**
changes to hackathon app to make it compatible with cornflow app templates.

### CODE REVIEW

File: hackathonbaobab2020/__init__.py [hunk 1]
```diff
@@ -1,3 +1,31 @@
+from typing import List, Dict
 from .core import *
 from .execution import *
 from .solver import *
+from .solver import solvers
```
Comment (ggsdc) [inline hackathonbaobab2020/__init__.py:5]: Does this two imports contradict each other?

File: hackathonbaobab2020/tests/tests.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import unittest
 import shutil
 from hackathonbaobab2020 import solve_zip, Experiment
+from hackathonbaobab2020 import HackathonApp
```
Comment (ggsdc) [inline hackathonbaobab2020/tests/tests.py:9]: Can't these two imports be done in one statement

File: hackathonbaobab2020/__init__.py (hunk 1 above)
Comment (pchtsp) [reply hackathonbaobab2020/__init__.py:5]: for some reason there was another `solvers` variable. I do not like importing *. I think I will import things explicitly.

File: hackathonbaobab2020/tests/tests.py (hunk 2 above)
Comment (pchtsp) [reply hackathonbaobab2020/tests/tests.py:9]: yes, my bad

**Pull request merged**